spec/features/Customer/show_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+RSpec.describe "Customer shoe page" do
+  before(:each) do
+    
+    @customer = Customer.create!(name: "Sally")
+    @supermarket = Supermarket.create!(name: "Corner Market", location: "123 Food St")
+    @item1 = @supermarket.items.create!(name: "banana", price: 1)
+    @item2 = @supermarket.items.create!(name: "apple", price: 2)
+    @customeritem = CustomerItem.create!(customer_id: @customer.id, item_id: @item1.id)
+  end
+
+    # Story 1
+
+    # As a visitor, 
+    # When I visit a customer show page,
+    # I see the customer's name,
+    # And I see its a list of its items
+    # including the item's name, price, and the name of the supermarket that it belongs to.
+
+    it "Show's a specific customer's details and items" do
+    
+
+        visit "/customers/#{@customer.id}"
+        
+        expect(page).to have_content(@customer.name)
+        expect(page).to have_content(@item1.name)
+        expect(page).to have_content(@item1.supermarket.name)
+        expect(page).to have_content(@item1.price)
```

Comment:
Since your view is set up with text labels (i.e. it can display as "Item Name: banana", or "Item Price: $1"), it is more meaningful to include these text labels in your assertions. For example, expecting your page to have `"Item Price: $4"` is more meaningful than just expecting "4" to appear somewhere on your page. False positives can also happen if there are other numbers on your page. What if your application has the company phone number at the bottom of the page, and that phone number includes a 4? Then, your test will pass even if you're not displaying the price. So, including the text label will make the assertion more specific!